src/test/java/teammates/test/cases/pagedata/InstructorFeedbackEditPageDataTest.java

```diff
@@ -64,7 +65,10 @@ public void allTests() {
         InstructorAttributes instructor = getInstructorFromBundle("instructor1OfCourse1");
         CourseDetailsBundle courseDetails = new CourseDetailsBundle(dataBundle.courses.get("typicalCourse1"));
 
-        data.init(fs, questions, questionHasResponses, studentList, instructorList, instructor,
+        List<FeedbackQuestionAttributes> templateQuestions = new ArrayList<>();
+        templateQuestions.add(dataBundle.feedbackQuestions.get("templateQn1InSampleSession1InCourse1"));
+
+        data.init(fs, questions, templateQuestions, questionHasResponses, studentList, instructorList, instructor,
```

Comment:
Yes. That is my intention, should I write a comment to be clearer? 

Cos I was thinking that it may be a bit repetitive to add all the questions from `feedbackSessionTeamEvaluationTemplate.json` into the `typicalDataBundle.json` and test everything so I just tested one question. What is your take?

Reply:
Is it possible that instead of getting all the questions from `typicalDataBundle.json` we get them from `feedbackSessionTeamEvaluationTemplate.json`?

So it wont require us to copy the questions. Also this way we can use[ I am not sure if the scope will allow us to use it here] and test the `getFeedbackSessionTemplateQuestions` function also.